### PR TITLE
ITM-527

### PR DESCRIPTION
**Pull request by Ayal-Ciobotaru**
Added Abrasion back in

### CODE REVIEW

File: swagger_server/itm/itm_action_handler.py [hunk 1]
```diff
@@ -118,6 +118,9 @@ def __successful_treatment(self, treatment: str, injury_name: str, location: str
                 return treatment == SupplyTypeEnum.SPLINT
             case InjuryTypeEnum.CHEST_COLLAPSE:
                 return treatment == SupplyTypeEnum.DECOMPRESSION_NEEDLE
+            case InjuryTypeEnum.ABRASION:
+                if 'face' in location:
```
Comment (nextcen-dgemoets) [inline swagger_server/itm/itm_action_handler.py:122]: Sorry, I forgot to add to the ticket the locations where they want the abrasion:
- left face, right face, right thigh, left thigh, right calf, left calf, right bicep, left bicep, right forearm, left forearm

Comment (Ayal-Ciobotaru) [reply swagger_server/itm/itm_action_handler.py:122]: Is the treatment for all abrasion locations a `pressure bandage`?

Comment (nextcen-dgemoets) [reply swagger_server/itm/itm_action_handler.py:122]: > Is the treatment for all abrasion locations a `pressure bandage`?

Yes.

**Review by nextcen-dgemoets — APPROVED** [review]
Looks good to me!

File: swagger_server/itm/itm_action_handler.py (hunk 1 above)
Comment (kaitlyn-sharo) [reply swagger_server/itm/itm_action_handler.py]: I was told back abrasion too, I thought

**Review by kaitlyn-sharo — APPROVED** [review]

**Pull request merged**